DQM/SiPixelPhase1TrackClusters/src/SiPixelPhase1TrackClusters.cc

```diff
@@ -142,6 +157,7 @@ void SiPixelPhase1TrackClusters::analyze(const edm::Event& iEvent, const edm::Ev
         histo[ONTRACK_SIZE      ].fill(double(cluster.size()  ), id, &iEvent);
         histo[ONTRACK_POSITION_B].fill(clustgp.z(),   clustgp.phi(),   id, &iEvent);
         histo[ONTRACK_POSITION_F].fill(clustgp.x(),   clustgp.y(),     id, &iEvent);
+	histo[ONTRACK_SIZE_VS_ETA].fill(clustgp.eta(), cluster.sizeY(), id, &iEvent);
```

Comment:
This is most likely the cause of the crash. `histo` is defined here
https://github.com/cms-sw/cmssw/blob/master/DQM/SiPixelPhase1Common/interface/SiPixelPhase1Base.h#L36

and its size is set here:
https://github.com/cms-sw/cmssw/blob/master/DQM/SiPixelPhase1Common/interface/SiPixelPhase1Base.h#L30

there is no guarantee that `histo.size() > ONTRACK_SIZE_VS_ETA`. In general, protected member functions are prone to this kind of problem which is why cms style guide says to only use private members and use member functions to protect access.